### PR TITLE
📱 Fix in-app notification & improve loading UX

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/components/ProgressIndicator.vue [hunk 1]
```diff
@@ -150,21 +150,24 @@ export default class ProgressIndicator extends Vue {
...
+    ];
+  }
+
   get progressContainer() {
     return {
-      class: [
-        'text-like-cyan-light',
-        { 'text-shade-gray': this.preset === ProgressIndicatorPreset.Thin },
-        'fill-like-cyan-light',
-      ],
```
Comment (nwingt) [inline src/components/ProgressIndicator.vue:167]: Using an object with key `class` is not flattened with `v-bind`, so I move the class array to separated computed prop

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/components/AuthRequiredView.vue [hunk 2]
```diff
@@ -8,8 +8,16 @@
       align="center"
       :text="loginLabel || $t('settings_page_content_with_auth_login_label')"
     />
-    <div class="flex justify-center mt-[24px]">
-      <ProgressIndicator v-if="walletIsLoggingIn" />
+    <div class="flex flex-col items-center mt-[24px]">
+      <template v-if="isLoadingAtStart || walletIsLoggingIn">
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/components/AuthRequiredView.vue:12]: Hmm, my purpose is not hiding connect button, this is solving the blinking UI before `walletIsLoggingIn` changes to `true`, how about `isLoadingStartImmediately`, like the `immediate: true` in watch?

[the remark this answers is not in the document]
Comment (williamchong) [reply src/components/AuthRequiredView.vue:12]: Sounds good to me

[the remark this answers is not in the document]
Comment (nwingt) [reply src/components/AuthRequiredView.vue]: Updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**